src/frontend/main.go

```diff
@@ -142,6 +143,29 @@ func main() {
 	log.Fatal(http.ListenAndServe(addr+":"+srvPort, handler))
 }
 
+func initJaegerTracing(log logrus.FieldLogger) {
+
+	svcAddr := os.Getenv("JAEGER_SERVICE_ADDR")
```

Comment:
This env var doesn't match what we had in the Java adservice. It was a bool; here it's an addr with !="" check.

I think we should unify the discovery?

Reply:
yes, I plan to modify the jave to match this one. I like this approach because it allows the user to name the endpoints.

Reply:
agreed this is better.